README.rst

```diff
@@ -425,6 +428,135 @@ view (only when using HTTP POST).
 The signal is emitted just before a successful response is returned,
 it is not sent if the response was not successful.
 
+Default Metrics
+---------------
+
+``Ping``
+~~~~~~~~
+
++--------------+-------------------------------------------------+
+|     name     |              ``Ping <Device>``                  |
```

Comment:
I think the name is not very useful to be shown here, however the description would be much more helpful.

Reply:
Agreed, so I think it is safe to remove the `name`  from this and all other metric' s table as well hence will proceed with it.